spec/controllers/v0/facilities/health_controller_spec.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 require 'rails_helper'
 
-RSpec.describe V0::Facilities::VaController, type: :controller do
+RSpec.describe V0::Facilities::HealthController, type: :controller do
```

Comment:
What do these controller specs provide that your request specs don't already satisfy? I find controller specs for APIs are usually not needed.